packages/x-charts-pro/package.json

```diff
@@ -66,6 +66,7 @@
     }
   },
   "devDependencies": {
+    "@mui/material": "next",
```

Comment:
The dev dependency here is the key to running the tests against the specific version of `@mui/material`.